adapters/MythicMobsAdapter/src/main/kotlin/com/caleb/typewriter/mythicmobs/entries/action/SpawnMobActionEntry.kt

```diff
@@ -0,0 +1,47 @@
+package com.caleb.typewriter.mythicmobs.entries.action
+
+import io.lumine.mythic.bukkit.BukkitAdapter
+import io.lumine.mythic.bukkit.MythicBukkit
+import me.gabber235.typewriter.adapters.Colors
+import me.gabber235.typewriter.adapters.Entry
+import me.gabber235.typewriter.adapters.modifiers.Help
+import me.gabber235.typewriter.entry.Criteria
+import me.gabber235.typewriter.entry.Modifier
+import me.gabber235.typewriter.entry.entries.ActionEntry
+import me.gabber235.typewriter.utils.Icons
+import org.bukkit.Location
+import org.bukkit.entity.Player
+import java.util.*
+
+
+@Entry("spawn_mythicmobs_mob", "Spawn a mob from mythicmobs", Colors.ORANGE, Icons.HANDSHAKE)
+class SpawnMobActionEntry(
+    override val id: String = "",
+    override val name: String = "",
+    override val criteria: List<Criteria> = emptyList(),
+    override val modifiers: List<Modifier> = emptyList(),
+    override val triggers: List<String> = emptyList(),
+    @Help("The mob's name")
+    private val mob: String = "",
+    @Help("The mob's level")
+    private val level: Double = 1.0,
+    @Help("Use player's location")
+    private val usePlayerLocation: Boolean = true,
+    @Help("Relative Location")
+    private var relativeLocation: Location,
```

Comment:
Instead of adding a boolean. Change the relative location to an optional `Optional<Location>`.

I know that it removes the option for absolute vs relative positions, but this is something that should be handled generally. To allow users to input relative locations with `~` or `^`. Please make a suggestion for this in the discord.